entry/src/main/resources/base/layout/ability_main.xml

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView
+    xmlns:ohos="http://schemas.huawei.com/res/ohos"
+    ohos:height="match_parent"
+    ohos:width="match_parent"
+    >
+
+   <DirectionalLayout
+       xmlns:ohos="http://schemas.huawei.com/res/ohos"
+       xmlns:custom="http://schemas.huawei.com/res/custom"
+       ohos:height="match_parent"
+       ohos:width="match_parent"
+       ohos:id="$+id:activity_main"
+       ohos:orientation="vertical">
+
+      <me.zhouzhuo.zzhoritontalprogressbar.ZzHorizontalProgressBar
+          ohos:height="60vp"
+          ohos:width="match_parent"
+          ohos:id="$+id:pb"
+          ohos:left_margin="20vp"
+          ohos:top_margin="20vp"
+          ohos:right_margin="20vp"
+          ohos:padding="20vp"
+          />
+
```

Comment:
Remove the unnecessary whitelines. Applies to all the java & xml files. Please check

Reply:
done